README.md

```diff
@@ -118,7 +128,7 @@ Once the library is loaded, you have to provide the environmental details along
  - extras - Optional simple map of additional key=value pairs you would like to pass to the authorization endpoint.
  - tokensAvailableHandler - function to be called when tokens are first available
```

Comment:
Not entirely sure about "first"; it seems to be a special case. If I understand it correctly, `getTokens` may be called at any time. 

In the other description it says:
> function to be called once tokens are available - either from the browser storage or newly fetched